library/Exceptions/AttributeException.php

```diff
@@ -13,10 +13,15 @@
 
 namespace Respect\Validation\Exceptions;
 
-class AttributeException extends NestedValidationException implements NonOmissibleException
+/**
+ * Exceptions to be thrown by the Attribute Rule.
+ *
+ * @author Henrique Moody <henriquemoody@gmail.com>
+ */
+final class AttributeException extends NestedValidationException implements NonOmissibleException
 {
-    public const NOT_PRESENT = 'not_present';
-    public const INVALID = 'invalid';
+    private const NOT_PRESENT = 'not_present';
+    private const INVALID = 'invalid';
```

Comment:
Shall we really do it? I think we should keep them public, at least for now. 

Also I think that later we will probably set these values from the validation